src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
I believe that better use `enum` for more convenient use.

Reply:
Optimized with `enum` type ⚙